crates/e2e/src/setup/services.rs

```diff
@@ -320,6 +328,7 @@ impl<'a> Services<'a> {
                 .fetch_one(db.deref_mut())
                 .await
                 .unwrap();
+            self.mint_block().await;
```

Comment:
What do you think about adding a comment explaining that this is required only for the `sync-to-blockchain` runloop mode.